.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -14,6 +14,7 @@ body:
       label: "Verified issue does not already exist?"
       description: "Please search to see if an issue already exists for the bug you encountered."
       options:
+        - "No, I did not search"

```

Comment:
lol its true,. i realized like a month later that there was only one selectable option